src/DynamoCoreWpf/Utilities/LibraryDragAndDrop.cs

```diff
@@ -29,18 +29,24 @@ internal void HandleMouseDown(Point position, NodeSearchElementViewModel node)
 
         internal void HandleMouseMove(FrameworkElement sender, Point currentPosition)
         {
-            if (isDragging || nodeViewModel == null)
+            if (isDragging)
```

Comment:
Why does this get removed? The `StartDrag` method does not guard against `NodeSearchElementViewModel == null` case.